contracts/resolvers/DelegatableResolver.sol

```diff
@@ -96,6 +97,15 @@ contract DelegatableResolver is
         emit Approval(node, operator, name, approved);
     }
 
+    /*
+     * Returns the owner address passed set by the Factory
+     * @return address The owner address
+     */
+    function owner() public view returns (address) {
+        require(_getArgAddress(0) != address(0), "Owner is not set");
```

Comment:
I was thinking if there's a way to prevent people interacting with a resolver not created by the factory

Reply:
Well, either the owner will be 0, in which case they won't have permission to, or it'll throw an error because the immutable data isn't present - which is also fine.